core/src/main/scala/com/mesosphere/package.scala

```diff
@@ -60,8 +60,8 @@ package mesosphere {
     *    // deleted immediately after. This happens every time the `logger` object is used to generate a log statement.
     * }}}
     */
-  case class LoggingArgs(args:(String, Any)*) {
-    def and(other:(String, Any)*): LoggingArgs = LoggingArgs(args ++ other:_*)
+  case class LoggingArgs(args: (String, Any)*) {
+    def and(other: (String, Any)*): LoggingArgs = LoggingArgs(args ++ other: _*)
```

Comment:
Thank you, i have somehow missed this when i first added the file. 
we can revisit this to automate in our scalafmt conf - may be when you bump your scalafmt plugin :)